readthedocs/projects/tasks.py

```diff
@@ -138,7 +138,7 @@ def sync_repo(self):
                 }
             )
             version_repo = self.get_vcs_repo()
-            version_repo.update()
+            version_repo.update(version=self.version)
```

Comment:
we need to pass the version here so that we can determine which PR should be fetched. there previous implementation fetched all the PR's all together regardless it was `open` or `closed`

Reply:
Seems like a good and small change 👍